activefires_pp/tests/test_spatiotemporal_alarm_filtering.py

```diff
@@ -328,15 +334,35 @@ def test_alarm_filter_runner_init(setup_comm, get_config):
                                     'restapi_url': 'https://xxx.smhi.se:xxxx'}
 
 
+@patch('activefires_pp.spatiotemporal_alarm_filtering.get_xauth_environment_variable')
+@patch('activefires_pp.spatiotemporal_alarm_filtering.read_config')
+@patch('activefires_pp.spatiotemporal_alarm_filtering.AlarmFilterRunner._setup_and_start_communication')
+def test_alarm_filter_runner_init_no_env(setup_comm, get_config, get_xauth):
+    """Test initialize the AlarmFilterRunner class."""
+    get_config.return_value = CONFIG_EXAMPLE
```

Comment:
Might be nicer with a fixture creating a file in `tmp_path`

Reply:
Ok, will consider that...

Reply:
Fixed.